BackupPC_report

```diff
@@ -23,10 +23,12 @@ my $bpcUser = 'backuppc';
 delete @ENV{qw(IFS CDPATH ENV BASH_ENV)};
 $ENV{PATH} = '/usr/bin';
 
-our ( $opt_a, $opt_s, $opt_w, $opt_help, $opt_man );
-GetOptions( "a", "s", "w=f", "help|?", "man" ) || pod2usage(2);
+our ( $opt_a, $opt_f, $opt_s, $opt_w, $opt_help, $opt_man );
+$opt_f = "report";
+GetOptions( "a", "f=s" => "report", "s", "w=f", "help|?", "man" ) || pod2usage(2);
```

Comment:
Would you clarify what does `=> "report"` do? I cannot find it in the Getopt::Long documentation.

Reply:
This was definitely not my intention. `=> "report"` should be `=> \$f_opt` for assignment

Reply:
Actually, it is not necessary. By default the variable $opt_X is a destination for the option X. So it can be shortened to
```suggestion
GetOptions( "a", "f=s", "s", "w=f", "help|?", "man" ) || pod2usage(2);
```

Reply:
okay changed and just using `"f=s"` now